.github/workflows/ci.yml

```diff
@@ -244,34 +244,6 @@ jobs:
       - name: Test (release)
         run: cross test --verbose --target=${{ matrix.target }} --release
 
-  features:
-    name: "Test cargo features (${{ matrix.simd }} × ${{ matrix.features }})"
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        simd:
-          - ""
-          - "avx512"
-        features:
-          - ""
-          - "--features std"
-          - "--features all_lane_counts"
```

Comment:
it may be useful to still test the other features and just delete the line with all_lane_counts.

Reply:
I actually changed `std` to a default feature (there was no reason to not test it), so I don't think there's any value testing without default features

Reply:
don't `#![no_std]` targets not enable the `std` feature? It still seems worth testing to me, and probably shouldn't be enabled by default.

Reply:
all the `std` feature actually does is enable testing `std_float`, which runs on every target.  the real issue is that on `no_std` targets that have libc, it calls into libc when it shouldn't (it works fine on no_std targets that don't have a libc)

Reply:
basically, we can test it on every target, we just can't include it in core